sqlmesh/core/model/__init__.py

```diff
@@ -14,6 +14,7 @@
 from sqlmesh.core.model.kind import (
     IncrementalByTimeRangeKind,
     IncrementalByUniqueKeyKind,
+    IncrementalUnsafeKind,
```

Comment:
is this the best name for this?

Reply:
Open to suggestions. I was definitely aiming at discouraging people from using it

Reply:
IncrementalManual
IncrementalUnmanaged

Reply:
oh, Unmanaged is pretty good